test/features/resource.author.feature

```diff
@@ -133,228 +103,359 @@ Feature: Resource
 
   @resource_author_07 @noworkflow
   Scenario: Add a resource with no datasets to a dataset with no resource
+    Given groups:
```

Comment:
@janette Do we need to create a group for this test?

Reply:
@fmizzell this allows the test to pass on sites that require a group for datasets